scripts/package-distribution.ps1

```diff
@@ -167,17 +180,13 @@ foreach ($Build in $AllBuilds) {
         if ($BuildBaseName -like "*openssl3*") {
             $Tls = "openssl3"
         }
-        $UbuntuVersion = "none"
-        $match = [regex]::Match($BuildBaseName, "ubuntu_(\d{4})")
-        if ($match.Success) {
-            $UbuntuVersion = $match.Groups[1].Value
-        }
+
         if ($BuildBaseName -like "*arm64_*") {
-            & $RootDir/scripts/make-packages.sh --output $DistDir --arch arm64 --tls $Tls --ubuntu $UbuntuVersion
+            & $RootDir/scripts/make-packages.sh --output $DistDir --arch arm64 --tls $Tls --xdp $UseXdp --time64 $Time64Distro
         } elseif ($BuildBaseName -like "*arm_*") {
-            & $RootDir/scripts/make-packages.sh --output $DistDir --arch arm --tls $Tls --ubuntu $UbuntuVersion
+            & $RootDir/scripts/make-packages.sh --output $DistDir --arch arm --tls $Tls --xdp $UseXdp --time64 $Time64Distro
         } else {
-            & $RootDir/scripts/make-packages.sh --output $DistDir --tls $Tls --ubuntu $UbuntuVersion # x64
+            & $RootDir/scripts/make-packages.sh --output $DistDir --tls $Tls --xdp $UseXdp --time64 $Time64Distro # x64
```

Comment:
like I said, this is only requirement for armhf for now, but I believe it's fine to keep it as-is, since all architectures of Ubuntu24.04 is already installing `libssl3t64` if you try to install `libssl3`